docs/snippets_test.py

```diff
@@ -406,7 +406,8 @@ def find_expected_outputs(snippet: str) -> List[str]:
                 expected.append(rest)
             else:
                 printing = False
-        elif line.startswith(start_key):
+        # Matches '# print', '# prints', '# print:', and '# prints:'
+        elif re.match('^# prints?:?\s*$', line):
```

Comment:
We should also match `#prints:` and similar. Throw in a `\s*` after the `#`.

Also, add tests exercising the cases.

Reply:
Yup. Not done yet as i realized this didn't fix the original error.

Reply:
Oh yeah the reason i didn't include the `\s*` was because it broke one of the tests. If we add it, this will be valid 
```
#prints something like (The other lines have spaces before comments begin except this one)
# transverse fields: [[-1, 1, -1], [1, -1, -1], [-1, 1, -1]]
# row j fields: [[1, 1, -1], [1, -1, 1]]
# column j fields: [[1, -1], [-1, 1], [-1, 1]]
```

Reply:
Not aesthetic, but if we want to give the user the option of doing it, i'll add it.

Reply:
It's fine to require that, because it causes a loud failure instead of a silent failure. But I agree that it could be nice to allow the content to be indented (but the indent should be consistent across the whole block of text). Out of scope for this PR.